### PR TITLE
Apposite 20487 d mod07 lak

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD07_LAK.md [hunk 1]
```diff
@@ -152,19 +145,19 @@
         _container = blogClient.GetContainerReference(_manifests);
```
Comment (VinayApposite) [inline Instructions/20487D_MOD07_LAK.md]: The use of the _ makes the section of code italics. Please replace with ASCII code or such.

File: Instructions/20487D_MOD07_LAK.md [hunk 2]
```diff
@@ -152,19 +145,19 @@
...
     }
     ```
-10. Locate **FinalizeFlight** method and paste the following code to upload a file to the container:
-    ```cs
+10. To upload a file to the container, locate the **FinalizeFlight** method, and then enter the following code:
+   ```cs
     await _container.CreateIfNotExistsAsync();
     CloudBlockBlob blockBlob = _container.GetBlockBlobReference($"{_manifests }.txt");
```
Comment (VinayApposite) [inline Instructions/20487D_MOD07_LAK.md]: The use of the _ makes the section of code italics. Please replace with ASCII code or such.

File: Instructions/20487D_MOD07_LAK.md [hunk 3]
```diff
@@ -177,113 +170,110 @@
...
 #### Task 1: Add a query to return related attractions from the itinerary service
 
-1. Click on **Keys** under **SETTINGS** section.
-2. Copy the **PRIMARY KEY** for the next step.
-3. Open **Command Line**.
-4. Run the following command to change directory to the starter project:
-    ```bash
-    cd [Repository Root]\Allfiles\Mod07\LabFiles\Lab2\Starter
+1. In the **SETTINGS** section, click **Keys**.
+2. For the next step, copy the **PRIMARY KEY** value.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD07_LAK.md]: Please verify my edit.

**Comment by barucht** [comment]
ready for QA

**Pull request merged**